uplink/ecclient/client.go

```diff
@@ -117,6 +120,9 @@ func (ec *ecClient) Put(ctx context.Context, limits []*pb.AddressedOrderLimit, p
 		}
 
 		if info.err != nil {
+			if !errs2.IsCanceled(info.err) {
```

Comment:
do you think it would be beneficial for us to report the number of nodes that were canceled? `segment_pieces_canceled` or something?

Reply:
Well you did say you can never have enough data